docs/manual/getting_started.rst

```diff
@@ -440,9 +544,9 @@ The content of the ``CMakeLists.txt`` is:
     "${CMAKE_SOURCE_DIR}/../../")
     endif()
 
-    # This is a convenience function, provided by the CycloneDDS package,
-    # that will supply a library target related the the given idl file.
-    # In short, it takes the idl file, generates the source files with
+    # This is a convenience function provided by the CycloneDDS package,
+    # that will supply a library target related to the given idl file.
+    # In short, it takes the idl file, and generates the source files with
     # the proper data types and compiles them into a library.
```

Comment:
`and` should not be added. And I think we should also standardise the usage of the Oxford comma. It could reduce some ambiguity in certain cases.
```suggestion
    # In short it takes the IDL file, generates the source files with
    # the proper data types, and then compiles them into a library.
```